mainwindow.cc

```diff
@@ -2509,15 +2509,7 @@ bool MainWindow::eventFilter( QObject * obj, QEvent * ev )
   if ( ev->type() == QEvent::ShortcutOverride
        || ev->type() == QEvent::KeyPress )
   {
-    // Handle Ctrl+H to show the History Pane.
     QKeyEvent * ke = static_cast<QKeyEvent*>( ev );
-    if ( ke->key() == Qt::Key_H && ke->modifiers() == Qt::ControlModifier )
```

Comment:
This part is already set via the .ui file. Since there is no conflict anymore, so this hack is useless.